apps/frontend/src/components/profile/Integrations.tsx

```diff
@@ -109,7 +109,7 @@ const Integrations: React.FC<IntegrationsProps> = ({ user }) => {
   return (
     <div className="mb-8">
       <h3 className="mb-4 text-xl font-semibold text-foreground">
-        Integrations
+        Connect your stack
```

Comment:
Just saw your conversation about the handling of this text for both onboarding and profile page. In these kind of scenarios you can just use a flag/prop to identify from where the component is being called. Or if you want to get more fancy you can check the pathname where the user is and then render your content based on that!

Reply:
actually i thought of this, but i guess this just makes it unnecessarily complex. keeping it simple, may be a good choice. also will change the color for the texts as mentioned

Reply:
here is how it will look - 

<img width="414" alt="Screenshot 2024-10-23 at 7 58 42 PM" src="https://github.com/user-attachments/assets/cc2bd9cf-24c8-4d4c-aa69-6eb79de401e7">
<img width="249" alt="Screenshot 2024-10-23 at 7 59 29 PM" src="https://github.com/user-attachments/assets/ee612417-d735-4590-b5b9-472cc267e59d">
<img width="296" alt="Screenshot 2024-10-23 at 7 59 45 PM" src="https://github.com/user-attachments/assets/fe198b34-09f1-4b74-8960-6f5a3e1f53ec">
<img width="405" alt="Screenshot 2024-10-23 at 8 00 20 PM" src="https://github.com/user-attachments/assets/ceae50c0-750d-40ba-a3f4-8aaa4971d875">


pushing them now !